libpyvinyl/Parameters/Collections.py

```diff
@@ -5,6 +5,38 @@
 
 from libpyvinyl.AbstractBaseClass import AbstractBaseClass
 from .Parameter import Parameter
+from pint.quantity import Quantity
+from pint.unit import Unit, UnitsContainer
+
+
+def quantity_encode(obj, primitives=False):
```

Comment:
type hint?

Reply:
I didn't put any because obj can be anything... so it does not seem to me very informative.
Any suggestion?